Templates/GitHubWebHook-CSharp/run.csx

```diff
@@ -0,0 +1,15 @@
+using System.Net;
+using System.Threading.Tasks;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
+{
+    log.Verbose($"C# HTTP trigger function processed a request. RequestUri={req.RequestUri}");
+
+    // Get request body
+    dynamic data = await req.Content.ReadAsAsync<object>();
```

Comment:
I was expecting the WebHooks library to have a class for the GitHub payload, but it doesn't look like it (@HenrikFrystykNielsen?).

So go with what you had initially with dynamic. We can always evolve later.